tests/fltest_screen.fish

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env fish
+# -*-  mode:fish; tab-width:4  -*-
+
+function fltest_screen
+
+    echo "Context: STY var is set to 'fishline_test'"
```

Comment:
Makes sense.  Probably worth documenting that somewhere (CONTRIBUTING.md?) but that's a separate issue.